source/Octopus.Tentacle/Kubernetes/KubernetesPodContainerRegistry.cs

```diff
@@ -36,7 +36,7 @@ public async Task<string> GetContainerImageForCluster()
 
             var tag = tagVersion?.ToString(2) ?? "latest";
 
-            return $"octopuslabs/k8s-workertools:{tag}";
+            return $"docker.packages.octopushq.com/octopusdeploy/k8s-agent-script-execution-image:{tag}";
```

Comment:
What testing have we done on this new image?

Reply:
I've tested the health check, raw yaml and helm step so far. I was hoping we could get this in so we can test it with different setups that we (as developers) are using. Then we can fix things as they come up. I can probably spin up a quick kustomize test too.

Reply:
Yep, we need to make sure that this container is going to work for all k8s steps

As mentioned in Slack, I also don't like the name of this image